doc/modules/model_persistence.rst

```diff
@@ -89,4 +88,28 @@ another architecture is not supported.
 
 If you want to know more about these issues and explore other possible
 serialization methods, please refer to this
-`talk by Alex Gaynor <https://pyvideo.org/video/2566/pickles-are-for-delis-not-software>`_.
+`talk by Alex Gaynor
+<https://pyvideo.org/video/2566/pickles-are-for-delis-not-software>`_.
+
+Interoperable formats
+---------------------
+
+For production and quality control needs, exporting the model in `Predictive
+Model Markup Language (PMML)
+<http://dmg.org/pmml/v4-4-1/GeneralStructure.html>`_ or `Open Neural Network
+Exchange <https://onnx.ai/>`_ format
+would be a better approach than using `pickle`.
+
+PMML is an extension of the `XML
+<https://fr.wikipedia.org/wiki/Extensible_Markup_Language>`_ document standard
+defined to represent data models together with the data used to generate them.
+Being human and machine readable,
+PMML is a good option for model validation on different platforms and
+long term archiving. On the other hand, as XML in general, its verbosity does
+not help in production when performance is critical.
```

Comment:
Maybe remove the last sentence. Typically a model would be loaded once, and then inference run several times. So the actual format XML vs something else, shouldn't matter much as long as inference is fast.

Reply:
For big models, (e.g. a large RF with deep trees) that XML format can lead to very high storage and CPU time for parsing the XML if I recall correctly.

Reply:
So, I will keep this way if you don't mind.